sssd_test_framework/topology_controllers.py

```diff
@@ -250,3 +251,64 @@ class IPATrustSambaTopologyController(IPATrustADTopologyController):
     """
 
     pass
+
+
+class IPATrustIPATopologyController(BackupTopologyController):
+    """
+    IPA trust IPA Topology Controller.
+    """
+
+    @restore_vanilla_on_error
+    def topology_setup(self, client: ClientHost, ipa: IPAHost, trusted: IPAHost) -> None:
+        if self.provisioned:
+            self.logger.info(f"Topology '{self.name}' is already provisioned")
+            return
+
+        # Add ipa-ipa trust COPR and update packages
+        self.logger.info("Adding COPR and updating packages")
+        ipa.conn.exec(["dnf", "copr", "enable", "abbra/wip-ipa-trust", "-y"])
+        client.conn.exec(["dnf", "copr", "enable", "abbra/wip-ipa-trust", "-y"])
+
+        ipa.conn.exec(["dnf", "update", "freeipa-server", "sssd-client", "-y"])
+        client.conn.exec(["dnf", "update", "sssd-client", "-y"])
+
+        # F40 sssd-kcm fails to start with 'Invalid option --genconf-section=kcm:'
+        ipa.conn.exec(["systemctl", "restart", "sssd-kcm"])
```

Comment:
Is this fixed/investigated already?

Reply:
Only relevant with install from COPR.

This will be removed later when feature is available and does not need to be installed from COPR, let's leave this conversation open.